bin/pasta_sync.py

```diff
@@ -101,6 +101,5 @@ def sync(config, prog, argv):
         config.update_ccache_mbox()
 
         # Update the mail thread cache
-        if not config.mbox_use_patchwork_id:
-            repo.mbox.load_threads()
-            repo.mbox.threads.update()
+        repo.mbox.load_threads()
+        repo.mbox.threads.update()
```

Comment:
Hmm... We have to keep that spot in mind for the future: In case of a patchwork analysis, we will never need the mail thread structures. Those structures are typically used for analysis that go beyond mapping patches. So it's useless to load thread structures if we do a patchwork analysis.

But for the moment, your correction is okay!